source/simulator/simulator_access.cc

```diff
@@ -706,6 +706,18 @@ namespace aspect
     return (simulator != nullptr);
   }
 
+
+
+  template <int dim>
+  bool
+  SimulatorAccess<dim>::simulator_is_past_initialization () const
+  {
+    return (simulator_is_initialized()
+            &&
```

Comment:
that logic is very confusing and points at inconsistent naming of functions/variables.